migrate/migrate.go

```diff
@@ -100,6 +113,54 @@ func (m *Migrator) migrateToV0_1_0(id uuid.UUID) error {
 	return nil
 }
 
+// for migration to new seperate Contract struct re:flipfest-21
+func (m *Migrator) migrateToV0_8_0(id uuid.UUID) error {
+	proj := model.InternalProject{
+		ID: id,
+	}
+
+	var accounts []*model.InternalAccount
+
+	err := m.store.GetAccountsForProject(proj.ID, &accounts)
+	if err != nil {
+		return errors.Wrap(err, "failed to get project accounts")
+	}
+
+	// loop through each account's single contract
+	// and create contract in a seperate Contract struct
+	for i, account := range accounts {
+		con := &model.Contract{
+			ProjectChildID: model.ProjectChildID{
+				ID:        uuid.New(),
+				ProjectID: proj.ID,
+			},
+			AccountIndex: i,
+			Title:        "[draft_contract]",
```

Comment:
I am slightly bothered that it is `"[draft_contract]"` here, but I cant think of a better alternative. Ideally the name would be equal to the contract, but that might be hard to achieve here.

Reply:
yes, if the contract has been deployed, its name would be in DeployedContracts. if not, need to extract contract name from DraftCode.